### PR TITLE
Implement missing namingFeature derived feature

### PR DESCRIPTION
**Pull request by adaussy**

### CODE REVIEW

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ConstraintUsageImpl.java [hunk 1]
```diff
@@ -431,4 +433,15 @@ public Object eInvoke(int operationID, EList<?> arguments) throws InvocationTarg
         return super.eInvoke(operationID, arguments);
     }
 
+    /**
+     * @generated NOT
+     */
+    @Override
+    public Feature namingFeature() {
+        OwningMembership owMembership = this.getOwningMembership();
+        if (owMembership instanceof RequirementConstraintMembership reqMembership && this.getOwnedReferenceSubsetting() != null) {
```
Comment (AxelRICHARD) [inline backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ConstraintUsageImpl.java:442]: You declare `reqMembership` but you never use it. could you fix that please?

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/PerformActionUsageImpl.java [hunk 2]
```diff
@@ -187,5 +190,14 @@ public EList<TextualRepresentation> getTextualRepresentation() {
         List<TextualRepresentation> textualRepresentation = new ArrayList<>();
         return new EcoreEList.UnmodifiableEList<>(this, SysmlPackage.eINSTANCE.getElement_TextualRepresentation(), textualRepresentation.size(), textualRepresentation.toArray());
     }
+    
+    @Override
```
Comment (AxelRICHARD) [inline backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/PerformActionUsageImpl.java:194]: Please add @generated NOT

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ConstraintUsageImpl.java (hunk 1 above)
Comment (adaussy) [reply backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ConstraintUsageImpl.java:442]: Done.

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/PerformActionUsageImpl.java (hunk 2 above)
Comment (adaussy) [reply backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/PerformActionUsageImpl.java:199]: done

**Pull request merged**